tianshou/data/buffer/base.py

```diff
@@ -232,7 +232,7 @@ def add(
         """
         # preprocess batch
         new_batch = Batch()
-        for key in set(self._input_keys).intersection(batch.keys()):
+        for key in batch.keys():
```

Comment:
could you change the above docstring accordingly? L223-L224 for example, not sure if there's any other place